docs/src/documentation/03-components/10-progress-indicators/skeleton/03-accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/skeleton/accessibility/
+---
+
+## Accessibility
+
+The Skeleton component has been designed with accessibility in mind, providing features that enhance the experience for users of assistive technologies during content loading.
+
+### Accessibility props
+
+The following props are available to improve the accessibility of your Skeleton component:
+
+| Name  | Type     | Description                                                            |
+| :---- | :------- | :--------------------------------------------------------------------- |
+| title | `string` | Provides an accessible description of the skeleton for screen readers. |
+
+The `title` prop is crucial for screen reader accessibility. When provided, it ensures that screen readers announce the loading state to users. Without it, users might not be aware that content is loading.
+
+### Automatic Accessibility Features
+
+The Skeleton component automatically implements the following accessibility features:
+
+- `aria-hidden="true"`: The SVG element that renders the skeleton is hidden from screen readers to prevent them from announcing the complex SVG structure.
+
+- When a `title` prop is provided, the component adds a visually hidden element containing the title text. This ensures screen readers announce the loading state without disrupting the visual appearance.
+
+- The component includes a subtle pulsing animation to indicate loading, but this animation is only applied when the user has not enabled reduced motion settings.
+
+These features ensure that the loading state is communicated effectively to screen reader users while maintaining a clean visual experience for all users.
+
+### Best practices
+
+- Always provide a `title` prop with a clear, descriptive message that informs users that content is loading. For example: "Loading profile information" or "Flight details loading".
+
+- Use proper ARIA attributes on the container that will display the actual content (not on the Skeleton itself):
```

Comment:
I added this part with `aria-busy` usage and examples at the bottom based on my understanding how the Skeletons should be used correctly and how other component libraries are recommending to use them. I also relied on this article https://adrianroselli.com/2020/11/more-accessible-skeletons.html